src/Models/CourseModel.php

```diff
@@ -46,29 +46,37 @@ public function addCourse(array $newCourse): string
             `name`,
             `notes`,
             `in_person`,
-            `remote`
+            `remote`,
+            `available_spaces_in_person`,
```

Comment:
should we keep this in line with the naming in the code base? i.e in_person_spaces?